apiserver/facades/client/action/action.go

```diff
@@ -270,6 +271,25 @@ func (a *ActionAPI) Enqueue(arg params.Actions) (params.ActionResults, error) {
 		return "", errors.Errorf("could not determine leader for %q", appName)
 	}
 
+	var operationName string
+	var receivers []string
+	for _, a := range arg.Actions {
+		if a.Receiver != "" {
+			receivers = append(receivers, a.Receiver)
+		}
+		if operationName == "" {
+			operationName = a.Name
+			continue
+		}
+		if operationName != a.Name {
+			operationName = "multiple actions"
+		}
+	}
+	operationId, err := a.model.EnqueueOperation(fmt.Sprintf("%v run on %v", operationName, strings.Join(receivers, ",")))
```

Comment:
would prefer the summary to be put into a variable and passed in.

Reply:
ok